tests/common.py

```diff
@@ -396,8 +421,14 @@ def async_fire_time_changed(
         # staggering to avoid thundering herd.
         utc_datetime = utc_datetime.replace(microsecond=500000)
 
-    timestamp = date_util.utc_to_timestamp(utc_datetime)
+    _async_fire_time_changed(hass, utc_datetime, fire_all)
+
 
+@ha.callback
+def _async_fire_time_changed(
+    hass: HomeAssistant, utc_datetime: datetime = None, fire_all: bool = False
```

Comment:
I think these two arguments are always passed internally, so no need for defaults
```suggestion
    hass: HomeAssistant, utc_datetime: datetime | None, fire_all: bool
```